pkg/synthetictests/disruption_test.go

```diff
@@ -0,0 +1,120 @@
+							Locator: "disruption/openshift-api connection/new",
+							Message: "reason/DisruptionSamplerOutageBegan DNS lookup timeouts began for disruption/openshift-api connection/new requests over new connections: Get \"https://api.ci-op-v352gg5m-99751.aws-2.ci.openshift.org:6443/apis/image.openshift.io/v1/namespaces/default/imagestreams\": dial tcp: lookup api.ci-op-v352gg5m-99751.aws-2.ci.openshift.org: i/o timeout (likely a problem in cluster running tests, not the cluster under test)",
+						},
+						From: now.Add(-31 * time.Second),
+						To:   now.Add(-30 * time.Second),
+					},
+					{
+						Condition: monitorapi.Condition{
+							Level:   0,
+							Locator: "disruption/cache-kube-api connection/new",
+							Message: "reason/DisruptionSamplerOutageBegan DNS lookup timeouts began for disruption/cache-kube-api connection/new GET requests over new connections: Get \"https://api.ci-op-v352gg5m-99751.aws-2.ci.openshift.org:6443/api/v1/namespaces/default?resourceVersion=0\": dial tcp: lookup api.ci-op-v352gg5m-99751.aws-2.ci.openshift.org: i/o timeout (likely a problem in cluster running tests, not the cluster under test)",
+						},
+						From: now.Add(-29 * time.Second),
+						To:   now.Add(-28 * time.Second),
+					},
+				},
+			},
+			want: []*junitapi.JUnitTestCase{
+				&junitapi.JUnitTestCase{
+					FailureOutput: &junitapi.FailureOutput{
+						Message: "reason/DisruptionSamplerOutageBegan DNS lookup timeouts began for disruption/openshift-api connection/new GET requests over new connections: Get \"https://api.ci-op-v352gg5m-99751.aws-2.ci.openshift.org:6443/apis/image.openshift.io/v1/namespaces/default/imagestreams\": dial tcp: lookup api.ci-op-v352gg5m-99751.aws-2.ci.openshift.org on 172.30.0.10:53: read udp 10.131.172.119:52726-\u003e172.30.0.10:53: i/o timeout (likely a problem in cluster running tests, not the cluster under test)",
+					},
+				},
+			},
+		},
+		{
+			name: "Detect real disruption just before DNS lookup errors",
+			args: args{
+				events: []monitorapi.EventInterval{
+					{
+						Condition: monitorapi.Condition{
+							Locator: "disruption/oauth-api connection/new",
+							Message: "reason/DisruptionBegan disruption/oauth-api connection/new stopped responding to GET requests over new connections: Get \"https://api.ci-op-v352gg5m-99751.aws-2.ci.openshift.org:6443/apis/oauth.openshift.io/v1/oauthclients\": dial tcp 3.133.199.188:6443: i/o timeout",
+						},
+						From: now.Add(-31 * time.Second),
+						To:   now.Add(-30 * time.Second),
+					},
+					{
+						Condition: monitorapi.Condition{
+							Locator: "disruption/cache-openshift-api connection/new",
+							Message: "reason/DisruptionBegan disruption/cache-openshift-api connection/new stopped responding to GET requests over new connections: Get \"https://api.ci-op-v352gg5m-99751.aws-2.ci.openshift.org:6443/apis/image.openshift.io/v1/namespaces/default/imagestreams?resourceVersion=0\": dial tcp 3.133.199.188:6443: i/o timeout",
```

Comment:
Based on test name it looks like this Message should be DNS disruption, not real.